### ISSUE TITLE
Oxygen 4.0 Alpha 1 – Margin / Padding bug

### ISSUE THREAD

**Issue by andreslotta** [soflyy/oxygen-bugs-and-features#2600]
**Describe the bug**
When I want to set a margin to an element the padding is set instead.

**Steps required to reproduce the bug:**
1. Add a heading element
2. Add a margin bottom of 32px (or whatever)
3. Save the page and check the source code on the frontend to see that padding is applied instead

**Screenshots:**
<img width="295" alt="Bildschirmfoto 2021-12-27 um 22 06 00" src="https://user-images.githubusercontent.com/12016014/147507293-83ed4535-9be2-4ee2-b597-ee870a304ef6.png">
<img width="795" alt="Bildschirmfoto 2021-12-27 um 22 06 22" src="https://user-images.githubusercontent.com/12016014/147507301-ecc57eab-c889-41ec-bcb5-ee33217c3bb7.png">

**Comment by adrien-robert** [comment]
They are already aware of this bug (heard this from Elijah from the live video from Permaslug)

**Comment by andreslotta** [comment]
> They are already aware of this bug (heard this from Elijah from the live video from Permaslug)

Yes. I saw it after opening the report. Now they have something to close after is has been fixed... ☺️

**Comment by Spellhammer** [comment]
Yep, we've reproduced and logged this. Thank you for posting the report!
